### PR TITLE
Load the period from config file

### PR DESCRIPTION
**Pull request by charlieegan3**
This will allow users to update the period on which data is sent by only
applying a new config.

Signed-off-by: Charlie Egan <charlieegan3@users.noreply.github.com>

### CODE REVIEW

File: pkg/agent/run.go [hunk 1]
```diff
@@ -84,6 +84,12 @@ func getConfiguration(ctx context.Context) (Config, *client.PreflightClient) {
 		}
 	}
 
+	// if period is set in the config, then use that
```
Comment (tfadeyi) [inline pkg/agent/run.go:87]: Looks good:+1: But could we do the assignment to Period outside `getConfiguration`?
Does the config has priority over the cli args?

Comment (charlieegan3) [reply pkg/agent/run.go]: I moved it to the main loop in fe707383d090dbe264fcafd4d2d541e790bd8e27. I think this is what we want. Other option is to set the default for the period to 0 and fail if it's not set, then they set it in the flag, or in config, where the flag takes priority.

Actually, I like that better.

Comment (charlieegan3) [reply pkg/agent/run.go]: 18e1ebcb3ef8a766a57637fb70d1a637bdc5f293

**Review by tfadeyi — APPROVED** [review]

**Review by dgellow — APPROVED** [review]
LGTM

File: cmd/agent.go [hunk 2]
```diff
@@ -41,7 +41,7 @@ func init() {
 		&agent.Period,
 		"period",
 		"p",
-		time.Hour,
+		0,
 		"Time between scans (given as XhYmZs).",
```
Comment (j-fuentes) [inline cmd/agent.go:45]:
```suggestion
		"Override time between scans in configuration file (given as XhYmZs).",
```

**Review by j-fuentes — APPROVED** [review]

File: cmd/agent.go (hunk 2 above)
Comment (charlieegan3) [reply cmd/agent.go]: Fixed in 0b00c186f8cdc4753036be85be1993de6c586e8f

**Pull request merged**